localization_manager/src/LocalizationTransitionTable.cpp

```diff
@@ -14,39 +14,38 @@
  * the License.
  */
 
-#include <ros/console.h>
-#include "localization_manager/LocalizationTransitionTable.h"
 
-namespace localizer
-{
-LocalizationTransitionTable::LocalizationTransitionTable(LocalizerMode mode) : mode_(mode)
+#include "localization_manager/LocalizationTransitionTable.hpp"
+
+
+namespace localization_manager
 {
-}
+
+LocalizationTransitionTable::LocalizationTransitionTable(LocalizerMode mode) : mode_(mode) {}
+
 LocalizationState LocalizationTransitionTable::getState() const
 {
   return state_;
 }
 
 void LocalizationTransitionTable::logDebugSignal(LocalizationSignal signal) const
 {
-  ROS_DEBUG_STREAM("LocalizationTransitionTable received unsupported signal of " << signal << " while in state "
-                                                                                 << state_);
+  RCLCPP_DEBUG_STREAM(rclcpp::get_logger("localization_manager::LocalizationTransitionTable"), "LocalizationTransitionTable received unsupported signal of "<< signal <<"while in state"<<state_);
 }
 
 void LocalizationTransitionTable::setAndLogState(LocalizationState new_state, LocalizationSignal source_signal)
 {
   if (new_state == state_)
   {
-    return;  // State was unchanged no need to log or trigger callbacks
+      return; //State was unchanged no need to log or trigger callbacks
   }
-  ROS_INFO_STREAM("LocalizationTransitionTable changed localization state from "
-                  << state_ << " to " << new_state << " because of signal " << source_signal << " while in mode "
-                  << mode_);
+  RCLCPP_INFO_STREAM(rclcpp::get_logger("localization_manager::LocalizationTransitionTable"), "LocalizationTransitionTable changed localization state from "<< state_ << " to "<< new_state <<" because of signal "<<source_signal << " while in mode "<<mode_);
```

Comment:
While a bit counter intuitive for the rclcpp::get_logger() methods to be controllable from the launch file the input should be the package name without extra namespaces.